Sources/PrivacyDashboard/PrivacyDashboardMode.swift

```diff
@@ -16,13 +16,25 @@
 //  limitations under the License.
 //
 
-/// Type of web page displayed
-public enum PrivacyDashboardMode: Equatable {
-
+/// Represents the type of web page displayed within the privacy dashboard flow.
```

Comment:
Thanks for taking time to add documentation. I think we could do it more inside BSK 👏